Automattic/wp-calypso
# Pricing page: Merge products and bundles section into one

#### Proposed Changes

* This PR will merge the Products and Bundles sections into one.
<img width="443" alt="Screen Shot 2022-12-19 at 12 32 22 PM" src="https://user-images.githubusercontent.com/56598660/208350325-36898be3-76c0-47b6-b5e5-c4d826b2b6cb.png">


#### Testing Instructions
1. Run this branch by following the steps below (or you can use the Jetpack Cloud live link commented on this PR)
    * Run `git fetch && git checkout update/merge-pricing-page-product-bundle`
    * Run `yarn start-jetpack-cloud`

2. Goto http://jetpack.cloud.localhost:3000/pricing or use the Jetpack cloud live link below and append `/pricing`
3. Confirm that the pricing page now has the new design

#### Pre-merge Checklist





Related to 1202858161751496-as-1203568228122298
